store.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/containers/storage/pkg/parsers"
 	"github.com/containers/storage/pkg/stringid"
 	"github.com/containers/storage/pkg/stringutils"
+	"github.com/docker/distribution/reference"
```

Comment:
We use `c/image/docker/reference` throughout (IIRC for somewhat historical reasons, the upstream package went through a painful API break at some point).